src/helics/shared_api_library/MessageFederateExport.cpp

```diff
@@ -19,11 +19,13 @@ SPDX-License-Identifier: BSD-3-Clause
 // random integer for validation purposes of endpoints
 static constexpr int EndpointValidationIdentifier = 0xB453'94C2;
 
-static inline void addEndpoint(helics_federate fed, std::unique_ptr<helics::EndpointObject> ept)
+static inline helics_endpoint addEndpoint(helics_federate fed, std::unique_ptr<helics::EndpointObject> ept)
 {
-    auto fedObj = reinterpret_cast<helics::FedObject*>(fed);
+    auto* fedObj = reinterpret_cast<helics::FedObject*>(fed);
```

Comment:
Why was this not required before?

Reply:
It isn't required, `auto` will correctly determine that it is a pointer type.  Clang-tidy suggests the `*` for clarity and readability,  I think mainly to make it obvious the fedObj is a pointer rather than an actual object.